extensions/cal-com-share-meeting-links/src/index.tsx

```diff
@@ -30,7 +30,7 @@ export default function Command() {
           key={item.id}
           title={item.title}
           accessories={getAccessories(item)}
-          subtitle={item.description}
+          keywords={item.length ? [item.length.toString()] : []}
```

Comment:
I don't think you should remove `subtitle` completely and replace it with keywords.

Reply:
That's so strange, I didn't have that line in my copy of the extension 🤔 Anyway, that's fixed now.

Reply:
Actually, I've just double checked and the version of the extension that's in the store now doesn't include the subtitle so maybe it's your version that's out of sync?

FWIW the subtitle looks a bit messy and isn't that useful, for me, as I already know what each event type is for, based its name.

With sub-titles (from my latest commit):

![CleanShot 2023-12-05 at 09 35 28@2x](https://github.com/raycast/extensions/assets/10284831/2e9466d6-5e46-4e30-afe5-f00635aebb17)

Without (from the store):

![CleanShot 2023-12-05 at 09 36 13@2x](https://github.com/raycast/extensions/assets/10284831/124b054c-4ef0-4a61-a341-d95e44c7af53)